src/StatsTools/convenience.jl

```diff
@@ -68,8 +19,10 @@ julia> res_df = DataFrame(res_w_errs) # results as DataFrame with lower an upper
 ─────┼────────────────────────────────────────────────────
    1 │ 1.01325  0.0173805  0.0183057  0.034042  0.0366713
 ```
+**Note:** This function will be deprecated soon. Use [`val_and_errs()`](@ref) instead.
 """
 function med_and_errs(p)
+    @warn "med_and_errs() will be deprecated soon. Use val_and_errs() instead!" maxlog=1
```

Comment:
Nitpick: the function is deprecated now and will be removed soon.

Reply:
Done